### PR TITLE
Fix no crs

### PR DESCRIPTION
**Pull request by olsen232**
## Description

Fix broken tests that check if we properly support datasets with no CRS
Fix issues preventing datasets with no CRS in different working copy types:

GPKG:  foreign key constraint
SQL Server: fails to roundtrip perfectly, `geometryCRS: CUSTOM:0` is added.

## Related links:

https://github.com/koordinates/kart/issues/855

## Checklist:

- [x] Have you reviewed your own change?
- [x] Have you included test(s)?
- [x] Have you updated the [changelog](https://github.com/koordinates/kart/blob/master/CHANGELOG.md)?

### CODE REVIEW

File: kart/tabular/working_copy/table_defs.py [hunk 1]
```diff
@@ -262,6 +262,24 @@ def __init__(self):
             ),
         )
 
+    ESPG_4326 = (
```
Comment (craigds) [inline kart/tabular/working_copy/table_defs.py:265]: ESPG -> EPSG

File: kart/tabular/working_copy/table_defs.py [hunk 2]
```diff
@@ -262,6 +262,24 @@ def __init__(self):
...
+    @classmethod
+    def init_table_contents(cls, sess):
+        sess.execute(
+            f"""
+            INSERT OR REPLACE INTO gpkg_spatial_ref_sys
+            (srs_name, srs_id, organization, organization_coordsys_id, definition, description)
+            VALUES
+            ('Undefined cartesian SRS', -1, 'NONE', -1, 'undefined', 'undefined cartesian coordinate reference system'),
+            ('Undefined geographic SRS', 0, 'NONE', 0, 'undefined', 'undefined geographic coordinate reference system'),
+            ('WGS 84 geodetic', 4326, 'EPSG', 4326, '{cls.ESPG_4326}', 'longitude/latitude coordinates in decimal degrees on the WGS 84 spheroid')
```
Comment (craigds) [inline kart/tabular/working_copy/table_defs.py:279]: i'd suggest parametrizing this properly - no real reason not to

**Review by craigds — APPROVED** [review]

File: kart/tabular/working_copy/table_defs.py (hunk 1 above)
Comment (olsen232) [reply kart/tabular/working_copy/table_defs.py]: Done

File: kart/tabular/working_copy/table_defs.py (hunk 2 above)
Comment (olsen232) [reply kart/tabular/working_copy/table_defs.py]: Done

**Pull request merged**